lib/v2/screens/app/interactor/viewmodels/app_bloc.dart

```diff
@@ -0,0 +1,31 @@
+import 'dart:async';
+import 'package:bloc/bloc.dart';
+import 'package:seeds/v2/screens/app/interactor/viewmodels/bloc.dart';
+import 'package:seeds/v2/screens/app/interactor/usecases/guardians_notification_use_case.dart';
+
+/// --- BLOC
+class AppBloc extends Bloc<AppEvent, AppState> {
+  StreamSubscription<bool?>? _hasGuardianNotificationPending;
+
+  AppBloc() : super(AppState.initial()) {
+    _hasGuardianNotificationPending = GuardiansNotificationUseCase()
+        .hasGuardianNotificationPending
+        .listen((value) => add(ShowNotificationBadge(value: value!)));
```

Comment:
why is ! needed? 
Once comments up top are completed `!` wont be needed

Reply:
done 👍